Fix:

error: implicit declaration of function 'radBuffersExit' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'radBuffersExitAndDestroy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'radBuffersInit' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'radProcessGetNotifyFD' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'radUtilsSleep' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://sourceforge.net/p/radlib/bugs/5/
--- src/radevents.c.orig	2010-02-13 07:55:01.000000000 -0600
+++ src/radevents.c	2025-08-17 22:01:58.000000000 -0500
@@ -47,6 +47,7 @@
 /*  ... Library include files
 */
 #include <radevents.h>
+#include <radprocess.h>
 
 
 /*  ... methods
--- src/radsqlite.c.orig	2011-04-01 16:11:11.000000000 -0500
+++ src/radsqlite.c	2025-08-17 23:02:12.000000000 -0500
@@ -46,6 +46,7 @@
 /*  ... Library include files
 */
 #include <radsqlite.h>
+#include <radsysutils.h>
 
 /*  ... Local include files
 */
--- src/radsystem.c.orig	2010-02-13 07:55:01.000000000 -0600
+++ src/radsystem.c	2025-08-17 22:32:32.000000000 -0500
@@ -47,6 +47,7 @@
 /*  ... Local header files
 */
 #include <radsystem.h>
+#include <radbuffers.h>
 
 #define _RAD_DBG_ENABLED        0
 #include <raddebug.h>
